app/scripts/models/auth_brokers/base.js

```diff
@@ -134,6 +150,19 @@ define(function (require, exports, module) {
       });
     },
 
+    /**
+     * Notify the browser that it should open pairing preferences
+     *
+     * @method openPairPreferences
+     * @returns {Promise} resolves when notification is sent.
+     */
+    openPairPreferences () {
+      if (this.hasCapability('supportsPairing')) {
+        const channel = this._notificationChannel;
+        return channel.send(channel.COMMANDS.PAIR_PREFERENCES);
+      }
+    },
```

Comment:
To make sure I understand correctly...is all this pairing-specific logic in the base auth-broker class, because it's used on a page without any `context=` parameter and which hence uses the default broker?  I guess I was mentally expecting to see it in one of the pairing-specific brokers, but having it here seems ok.

Reply:
> guess I was mentally expecting to see it in one of the pairing-specific brokers, but having it here seems ok.

Exactly, because we ask users to navigate directly to `/pair` without any contexts.